sdk/appcenter-crashes/src/main/java/com/microsoft/appcenter/crashes/Crashes.java

```diff
@@ -1148,34 +1122,14 @@ private UUID saveErrorLogFiles(Throwable throwable, ManagedErrorLog errorLog) th
         String filename = errorLogId.toString();
         AppCenterLog.debug(Crashes.LOG_TAG, "Saving uncaught exception.");
         File errorLogFile = new File(errorStorageDirectory, filename + ErrorLogHelper.ERROR_LOG_FILE_EXTENSION);
+
+        /* Set stacktrace to error log. */
+        errorLog.getException().setStackTrace(getStackTraceString(throwable));
```

Comment:
Will this be sent in the JSON? We should clean it up before sending.

Reply:
Do you mean that this report will be sent on the portal? If so, then I suggest that the report for sending is generated [here](https://github.com/microsoft/appcenter-sdk-android/blob/1a04d59f0f4382a67e6ee2f02bbf0a1bb9589733/sdk/appcenter-crashes/src/main/java/com/microsoft/appcenter/crashes/Crashes.java#L916) and stack trace info adding to error report only once.

Reply:
Yes this field would be sent in json but ignored by backend and can be big, it duplicates the structured stack trace.
So there should be a place in the code that unset the field when sending (at latest before enqueuing in channel).